contents/customers/assemblyai.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Why AssemblyAI switched from Mixpanel to PostHog
+customer: AssemblyAI
+logo: ../images/customers/assemblyai/logo.png
+featuredImage: ../images/customers/assemblyai/featured.png
+industries:
+    - API Platform
+users:
+    - Leadership
+    - Marketing
+    - Engineering
+toolsUsed:
+    - Experimentation
+    - Cloud-hosted
+    - Insights
+---
+
+[AssemblyAI](https://www.assemblyai.com/?utm_source=posthog&utm_campaign=case_study) is an API platform which provides powerful AI models to transcribe and understand speech. As a young company in the cutting-edge field of AI modeling, AssemblyAI enjoys an all-hands-on-deck environment where teams wear many hats and have wide remits. In such a fast-moving culture, having immediate access to reliable data is essential for decision-making about the future of the product. 
+
+To enable this, AssemblyAI had previously used Mixpanel, but as the company grew and users flooded in, Mixpanel’s pricing model made it increasingly expensive. 
```

Comment:
```suggestion
AssemblyAI previously used Mixpanel to enable this but, as the company grew and users flooded in, Mixpanel’s pricing model made it increasingly poor value. 
```

Maybe unaffordable rather than "poor value"? Up to you.